config.yml

```diff
@@ -93,10 +93,10 @@ params:
     js_bundle_hash:                 "sha384-2jsvnZnyYAg6xCGTSIdBPoEt3xP64FauXZlBK9PAYsneWVkqXT/5keQRlbdfXkCL"
     jquery:                         "https://code.jquery.com/jquery-3.5.1.slim.min.js"
     jquery_hash:                    "sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
-    css_swiper:                     "https://cdn.jsdelivr.net/npm/swiper@4.5.1/dist/css/swiper.min.css"
-    css_swiper_hash:                "sha384-3ycIs75n6btZktoCV/zcWo54Eq+bnjaJ9iwOISgNPQIXRSmrkf4XP1iHsWNq0jPB"
-    js_swiper:                      "https://cdn.jsdelivr.net/npm/swiper@4.5.1/dist/js/swiper.min.js"
-    js_swiper_hash:                 "sha384-llVNZVxgabZyf5ZeGs3m2QcNbEE0UK1PBKM6ZoJmWK5BuBpqZUXpN1nWXd0SrAC5"
+    css_swiper:                     "https://unpkg.com/swiper@8.3.0/swiper-bundle.min.css"
```

Comment:
TODO:
* Switch to 8.3.1
* Revert unpkg.com and use https://cdn.jsdelivr.net/npm/swiper@8.3.1/swiper-bundle.min.css